UltraStar Play/Assets/Scenes/Sing/PlayerController/RecordedNote.cs

```diff
@@ -14,6 +14,14 @@ public class RecordedNote
     public Sentence Sentence { get; set; }
     public Note TargetNote { get; set; }
 
+    public bool IsTargetNoteHit
```

Comment:
this code is only used in PlayerNoteRecorder. The RecordedNote class should probably be kept as simple and "dumb" as possible (like a Java bean). I'd suggest moving this function to PlayerNoteRecorder or some Utility/Support class?